sdks/python/apache_beam/runners/dask/dask_runner.py

```diff
@@ -92,6 +105,21 @@ def _add_argparse_args(cls, parser: argparse.ArgumentParser) -> None:
         default=512,
         help='The number of open comms to maintain at once in the connection '
         'pool.')
+    partitions_parser = parser.add_mutually_exclusive_group()
```

Comment:
Is this worth calling out in our CHANGES.md?

Reply:
Yeah, I think that makes sense.